src/build_tools.md

```diff
@@ -0,0 +1,128 @@
+# Build Time Tooling
+
+This section covers "build-time" tooling, or code that is run prior to compiling a crate's source code.
+Conventionally, build-time code lives in a `build.rs` file and is commonly referred to as a "build script".
```

Comment:
Sorry to catch this this late but generally we strive to use backticks <code>``</code> for code /symbols or external commands and for other text we use different form of emphasis (italics or bold). So I would change the method of emphasis for the file/language/crate/website names and use the backticks only for symbols and language keywords (`extern`)